doc/src/vpr/file_formats.rst

```diff
@@ -585,6 +585,10 @@ The io pad is set to inpad mode and is driven by the inpad:
         </block>
     ...
 
+.. note:: ``.net`` files may be outputted at two stages:
+          - After packing is completed, the packing results will be outputted.
+          - After analysis is completed, updated packing results will be outputted. This is due to that VPR router may swap pin mapping in packing results for optimizations. In such cases, packing results are synchronized with routing results. The outputted ``.net`` file will have a postfix of ``.post_routing`` as compared to the original packing results. It could happen that VPR router does not apply any pin swapping and the two ``.net`` files are the same. But we recommend to use the post-analysis ``.net`` file as the final packing results.
```

Comment:
Now that we have two `.net` outputs, what is the theory of loading netlist?  When does VPR ever read in the `.post_routing`?